docs/quickstart-minikube.md

```diff
@@ -259,12 +259,13 @@ quickstart-wordpress   1         1         1            1           2m
 
 1. Run the following command to open WordPress in your browser:
     ```
-    open http://$(minikube ip):31215/admin
+    open http://$(minikube ip):$(kubectl get service quickstart-wordpress -o jsonpath={.spec.ports[?\(@.name==\"http\"\)].nodePort}) 
```

Comment:
Did we lose the /admin path part?

Reply:
GOOD CATCH!! I accidentally overwrote that bit. Updated the PR.